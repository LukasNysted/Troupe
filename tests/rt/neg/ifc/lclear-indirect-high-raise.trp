(* negative ; receiving below the level of the pc at the raise time *)
let val _ = _setProcessDebuggingName ("main") 
    val _ = send (self (), 1)
in 
if true raisedTo `{secret}` then 
  let val c = raisembox (`{secret}`)
   in rcv (`{}`, `{secret}`, [hn x => x])
  end 
else 
  ()
end    