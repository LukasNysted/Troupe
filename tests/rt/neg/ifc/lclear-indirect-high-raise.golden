2023-03-17T21:23:24.046Z [RTM] [32minfo[39m: Skipping network creation. Observe that all external IO operations will yield a runtime error.
Runtime error in thread [main]861e6f6a-6f0d-4945-b036-e41d736b1cf3@{}%{}
>> PC level at the time of raising the mailbox clearance is too sensitive for this receive
 | receive lower bound: {}
 | pc level at the time of receive: {secret}
 | pc level at the time of raise: {secret}
